app/scheduled/circulars/uploadTar.ts

```diff
@@ -0,0 +1,124 @@
+/*!
+ * Copyright © 2023 United States Government as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ * All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { PutObjectCommand, S3Client } from '@aws-sdk/client-s3'
+import { createReadableStreamFromReadable } from '@remix-run/node'
+import { Readable } from 'stream'
+import type { Pack } from 'tar-stream'
+import { pack as tarPack } from 'tar-stream'
+
+import { getEnvOrDie } from '~/lib/env.server'
+import type { Circular } from '~/routes/circulars/circulars.lib'
+import { formatCircular } from '~/routes/circulars/circulars.lib'
+
+interface TarContextObject {
```

Comment:
```suggestion
interface TarContext {
```

Reply:
COUREY: Add context here so it's {context: {pack.....}}